### PR TITLE
Update android sample with latest wallet core build

### PR DESCRIPTION
**Pull request by hewigovens**
Fixes https://github.com/trustwallet/wallet-core/issues/1577

### CODE REVIEW

**Review by catenocrypt — APPROVED** [review]

**Comment by catenocrypt** [comment]
The Android CI always builds the android sample app.  How come this issue was not caught?

**Comment by hewigovens** [comment]
it installed old version wallet core

**Comment by hewigovens** [comment]
@catenocrypt docs on developer also needs to update: https://github.com/trustwallet/developer/issues/184

File: samples/android/build.gradle [hunk 1]
```diff
@@ -4,22 +4,25 @@ buildscript {
...
 allprojects {
     repositories {
         google()
-        jcenter()
-        
+        mavenCentral()
+        maven {
+            url = uri("https://maven.pkg.github.com/trustwallet/wallet-core")
+            credentials {
+                username = project.findProperty("gpr.user") as String?: System.getenv("GITHUB_USER")
```
Comment (vikmeup) [inline samples/android/build.gradle:22]: this is needed to fetch dependency from walle-core?

**Review by vikmeup — APPROVED** [review]

File: samples/android/build.gradle (hunk 1 above)
Comment (hewigovens) [reply samples/android/build.gradle:22]: yes, GitHub package still needs authentication

**Pull request merged**